staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch.go

```diff
@@ -43,7 +43,10 @@ const (
 	replaceDirective = "replace"
 	mergeDirective   = "merge"
 
+	replaceKeysStrategy = "replaceKeys"
+
 	deleteFromPrimitiveListDirectivePrefix = "$deleteFromPrimitiveList"
+	replaceKeysDirective                   = "$" + replaceKeysStrategy
```

Comment:
Why is this `-Directive` variable prefixed with a dollar sign when the other Directive variables are not?